core/integration/ddsched/plugin.go

```diff
@@ -234,9 +234,9 @@ func (p *Plugin) ObjectStack(data interface{}) (stack map[string]interface{}) {
 				Error("failed to perform DD scheduler PartitionInitialize")
 			return
 		}
-
+		pollingTimeout := 15 * time.Second
```

Comment:
Please read this value from a variable `dd_polling_timeout` (you can access `varStack` from this context). If the variable is not defined, we default to `30s`. The reason for this is that depending on the workflow structure, different DD timeouts can be appropriate.